src/features/defi/providers/univ2/components/UniV2Manager/Deposit/components/Deposit.tsx

```diff
@@ -201,28 +201,28 @@ export const Deposit: React.FC<DepositProps> = ({
         // While the naive approach would be to think both assets approve() calls are going to result in the same gas estimation,
         // this is not necesssarly true. Some ERC-20s approve() might have a bit more logic, and thus require more gas.
         // e.g https://github.com/Uniswap/governance/blob/eabd8c71ad01f61fb54ed6945162021ee419998e/contracts/Uni.sol#L119
-        const asset0EstimatedGasCrypto =
+        const asset0FeeData =
           assetId0 !== ethAssetId && (await getApproveFeeData(asset0ContractAddress!))
-        const asset1EstimatedGasCrypto =
+        const asset1FeeData =
```

Comment:
Can get `txFee` on the right side of the short circuit since this is the only property we use from `FeeData` here